core/scripts/ocr2vrf/util.go

```diff
@@ -0,0 +1,177 @@
+	keyIDBytes := decodeHexTo32ByteArray(keyID)
+
+	dkg, err := dkgContract.NewDKG(common.HexToAddress(dkgAddress), e.Ec)
+	helpers.PanicErr(err)
+
+	tx, err := dkg.AddClient(e.Owner, keyIDBytes, common.HexToAddress(clientAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func removeClientFromDKG(e helpers.Environment, dkgAddress string, keyID string, clientAddress string) {
+	keyIDBytes := decodeHexTo32ByteArray(keyID)
+
+	dkg, err := dkgContract.NewDKG(common.HexToAddress(dkgAddress), e.Ec)
+	helpers.PanicErr(err)
+
+	tx, err := dkg.RemoveClient(e.Owner, keyIDBytes, common.HexToAddress(clientAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func setDKGConfig(e helpers.Environment, dkgAddress string, c dkgSetConfigCommands) {
+	oracleIdentities := toOraclesIdentityList(helpers.ParseAddressSlice(c.onchainPubKeys),
+		helpers.ParseStringSlice(c.offchainPubKeys), helpers.ParseStringSlice(c.configPubKeys),
+		helpers.ParseStringSlice(c.peerIDs), helpers.ParseStringSlice(c.transmitters))
+
+	ed25519Suite := edwards25519.NewBlakeSHA256Ed25519()
+	var signingKeys []kyber.Point
+	for _, signingKey := range helpers.ParseStringSlice(c.dkgSigningPubKeys) {
+		signingKeyBytes, err := hex.DecodeString(signingKey)
+		helpers.PanicErr(err)
+		signingKeyPoint := ed25519Suite.Point()
+		signingKeyPoint.UnmarshalBinary(signingKeyBytes)
+		signingKeys = append(signingKeys, signingKeyPoint)
+	}
+
+	altbn128Suite := &altbn_128.PairingSuite{}
+	var encryptionKeys []kyber.Point
+	for _, encryptionKey := range helpers.ParseStringSlice(c.dkgEncryptionPubKeys) {
+		encryptionKeyBytes, err := hex.DecodeString(encryptionKey)
+		helpers.PanicErr(err)
+		encryptionKeyPoint := altbn128Suite.G1().Point()
+		encryptionKeyPoint.UnmarshalBinary(encryptionKeyBytes)
+		encryptionKeys = append(encryptionKeys, encryptionKeyPoint)
+	}
+
+	keyIDBytes := decodeHexTo32ByteArray(c.keyID)
+
+	offchainConfig, err := dkg.OffchainConfig(dkg.EncryptionPublicKeys(encryptionKeys), dkg.SigningPublicKeys(signingKeys), &altbn_128.G1{}, &trivialTranslation{})
+	helpers.PanicErr(err)
+	onchainConfig, err := dkg.OnchainConfig(dkg.KeyID(keyIDBytes))
+	helpers.PanicErr(err)
+
+	_, _, f, onchainConfig, offchainConfigVersion, offchainConfig, err := confighelper.ContractSetConfigArgsForTests(
+		toDurationInSeconds(c.deltaProgress), toDurationInSeconds(c.deltaResend), toDurationInSeconds(c.deltaRound), toDurationInSeconds(c.deltaGrace), toDurationInSeconds(c.deltaStage),
+		uint8(c.maxRounds), helpers.ParseIntSlice(c.schedule), oracleIdentities, offchainConfig, toDurationInSeconds(c.maxDurationQuery), toDurationInSeconds(c.maxDurationObservation),
+		toDurationInSeconds(c.maxDurationReport), toDurationInSeconds(c.maxDurationAccept), toDurationInSeconds(c.maxDurationTransmit), int(c.f), onchainConfig)
```

Comment:
formatting here?

Reply:
yeah making it one line per arg is probably better